.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # dependencies
 node_modules
+bower_components
```

Comment:
since bower is officially deprecated, why not be possible install it from npm registry?

```
npm install polymer-microlink
```

another thing, is it recommended  or possible load polymer scripts from an CDN?

Reply:
I tried, but some component dependencies are not in npm yet... there are only two critical dependencies, i'll take a look to see if we can replace them to use something from npm ;)

Reply:
even if the dependencies are not on npm, they can be installed from a GitHub repository as well. You can specify the branch too!